packages/modules/devices/fems/bat.py

```diff
@@ -20,7 +20,8 @@ def update(self, session: Session) -> None:
         else:
             data = "ess2"
         response = session.get(
-            "http://" + self.ip_address + ":8084/rest/channel/"+data+"/(Soc|DcChargeEnergy|DcDischargeEnergy)",
+            "http://" + self.ip_address + ":8084/rest/channel/(" + data + "|_sum)/" +
```

Comment:
Sorry, but this change breaks queries on older FEMS versions.

At least on my FEMS v 2022.12.5 a query with Regex in any "segment" other than the last one is NOT supported. FEMS just returns:

`requests.exceptions.HTTPError: 404 Client Error: Not Found for url: http://<IP>:8084/rest/channel/(meter0%7C_sum)/(ActivePower.*%7CVoltageL.%7CFrequency%7CGrid.+ActiveEnergy)`

I also see the optimization done here. But I might not be the only one with older version. Actually I have no control on if and when Fenecon updates my FEMS.

So I have no other idea but reverting this!